spec/index.bs

```diff
@@ -266,10 +240,38 @@ dictionary GPUObjectDescriptorBase {
         The initial value of {{GPUObjectBase/label|GPUObjectBase.label}}.
 </dl>
 
+## Invalid Internal Objects &amp; Contagious Invalidity ## {#invalidity}
```

Comment:
Did you coin this term? Is there going to be more added later to explain it, or does it just refer to the `If the device that owns an object is lost, the object becomes invalid` line?

Reply:
"Contagious Invalidity" is an update of the term we used to use "Contagious Internal Nullability" to match the current wording ("invalid" objects).

Yes, I'm planning to write more on the topic along and formalize it more.

Reply:
(It refers to the fact that if you use an invalid object, like a GPUBuffer, in the creation of a new object, like a GPUBindGroup, the new object is also invalid)